stablehlo/reference/Ops.cpp

```diff
@@ -618,6 +770,58 @@ SmallVector<Tensor> eval(
       Tensor runtimeOperand = scope.find(realOp.getOperand());
       Tensor runtimeResult = evalRealOp(runtimeOperand, realOp.getType());
       scope.add(op.getResults(), {runtimeResult});
+    } else if (auto reduceOp = dyn_cast<ReduceOp>(op)) {
+      SmallVector<Tensor> runtimeInputs = scope.find(reduceOp.getInputs());
+      SmallVector<Tensor> runtimeInitValues =
+          scope.find(reduceOp.getInitValues());
+      SmallVector<TensorType> resultTypes;
+      for (auto resultType : reduceOp.getResultTypes())
+        resultTypes.push_back(resultType.cast<TensorType>());
+      SmallVector<Tensor> runtimeResults = evalReduceOp(
+          runtimeInputs, runtimeInitValues, Axes(reduceOp.getDimensions()),
+          reduceOp.getBody(), scope, resultTypes);
+      scope.add(op.getResults(), {runtimeResults});
+    } else if (auto reduceWindowOp = dyn_cast<ReduceWindowOp>(op)) {
+      SmallVector<Tensor> runtimeInputs =
+          scope.find(reduceWindowOp.getInputs());
+      SmallVector<Tensor> runtimeInitValues =
+          scope.find(reduceWindowOp.getInitValues());
+      SmallVector<TensorType> resultTypes;
+      for (auto resultType : reduceWindowOp.getResultTypes())
+        resultTypes.push_back(resultType.cast<TensorType>());
+      SmallVector<int64_t> windowStrides =
```

Comment:
Use the constructor of `SmallVector` instead of assignment.

```
    SmallVector<int64_t> windowStrides(runtimeInputs[0].getRank(), 1);
      if (reduceWindowOp.getWindowStrides().has_value())
        windowStrides = llvm::to_vector(
            (*reduceWindowOp.getWindowStrides()).getValues<int64_t>());
```